### DISCUSSION TITLE
Blank page after deployment, but build works locally

### DISCUSSION THREAD

**Discussion by leedsrising** [vercel/community#2564] [Help]
### Summary

Im seeing a blank page when visiting my build https://mysimpleco-poefsg5pf-leedsrising.vercel.app/

but when running locally (npm start), running my build locally (with ```vercel dev``` or ```serve -s``` in my build folder) I'm seeing my app in its intended form

https://github.com/vercel/next.js/discussions/36051#discussioncomment-4806015 ack'ed a similar issue with next.js. I was doing some jank with both next.js + create react app, but npm uninstalled next.js to try and get this to work

also tried adding ```"homepage": "."``` in my package.json per a StackOverflow recommendation

### Example

https://mysimpleco-poefsg5pf-leedsrising.vercel.app/

**Comment by leedsrising** [comment]
Looks like the assets in my vercel deploy also don't match my build folder https://jmp.sh/XfaTPZNp+/App.js+%E2%80%94+mysimpleco+2023-05-23+at+9.51.16+PM.png

**Comment by amyegan** [comment]
When I view page source on the site your shared, it looks like a React SPA page template. I wonder if there is a package or asset available on your local environment that is not being included in the deployment. Or maybe there is a difference in build  or install commands. 

If you share a minimal reproduction, or your repo if it is public, then someone here may be able to recognize the cause of this problem.

**Comment by leedsrising** [comment]
> When I view page source on the site your shared, it looks like a React SPA page template.

How do you mean exactly? You're not seeing a blank page?

And just made the repo public https://github.com/leedsrising/mysimpleco

**Comment by amyegan** [comment]
Thanks for sharing the repo! I was able to deploy it and see a form on the homepage: https://mysimpleco-khaki.vercel.app/

I chose the "Create React App" framework preset for my project's [Build and Development Settings](https://vercel.com/docs/concepts/deployments/configure-a-build#build-and-development-settings). If you have any overrides or are using a different preset, I recommend trying it with "Create React App" selected.

**Comment by leedsrising** [comment]
Interesting, I still can't get the deploy to work

Ive turned off overrides, and I am using the create react app framework

https://mysimpleco-6ju606kxm-leedsrising.vercel.app/ is the most recent redeploy on my side
